### DISCUSSION TITLE
Auto-promotion sur ma cantine

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1045] [Idées]
Que pensez-vous de rendre la brochure ma cantine accessible en téléchargement quelque part sur la Landing page.
On peut la mettre sur Gitbook, mais je pense qu'elle perdrait un peu en visibilité. Un peu d'auto-promotion, c'est cool 💪. 
Cette demande se base sur quelques retours utilisateurs (tête de réseaux surtout) et sur le fait que ce document permet d'avoir une vision très synthétique de la plateforme. 

[ma cantine brochure.pdf](https://github.com/betagouv/ma-cantine/files/8042427/ma.cantine.brochure.pdf)

**Comment by hfroot** [comment]
Merci pour la suggestion - est-ce que tu peux nous donner plus de contexte sur quoi les utilisateurs veulent faire avec ce brochure ?
Pour clarifier - je suis curieuse de savoir pourquoi (ou si) le format PDF est favorisé

**Reply by gauthierandre** [reply]
En gros la brochure permet d'avoir une présentation synthétique de la plateforme. C'est à l'attention des têtes de réseaux, des personnes qui souhaitent connaître notre produit sans l'utiliser ou l'utiliser dans l'immédiat. On la distribue en salon etc.
